Doc/library/sys.rst

```diff
@@ -1065,18 +1065,18 @@ always available.
    +----------------------------------------+-----------------------------------------------+
    | Attribute                              | Explanation                                   |
    +========================================+===============================================+
-   | :const:`bits_per_digit`                | number of bits held in each digit.  Python    |
+   | :attr:`!bits_per_digit`                | number of bits held in each digit.  Python    |
    |                                        | integers are stored internally in base        |
    |                                        | ``2**int_info.bits_per_digit``                |
    +----------------------------------------+-----------------------------------------------+
-   | :const:`sizeof_digit`                  | size in bytes of the C type used to           |
+   | :attr:`!sizeof_digit`                  | size in bytes of the C type used to           |
    |                                        | represent a digit                             |
    +----------------------------------------+-----------------------------------------------+
-   | :const:`default_max_str_digits`        | default value for                             |
+   | :attr:`!default_max_str_digits`        | default value for                             |
    |                                        | :func:`sys.get_int_max_str_digits` when it    |
    |                                        | is not otherwise explicitly configured.       |
    +----------------------------------------+-----------------------------------------------+
-   | :const:`str_digits_check_threshold`    | minimum non-zero value for                    |
+   | :attr:`!str_digits_check_threshold`    | minimum non-zero value for                    |
    |                                        | :func:`sys.set_int_max_str_digits`,           |
    |                                        | :envvar:`PYTHONINTMAXSTRDIGITS`, or           |
    |                                        | :option:`-X int_max_str_digits <-X>`.         |
```

Comment:
```rest
   .. attribute:: int_info.bits_per_digit

      The number of bits held in each digit.
      Python integers are stored internally in base ``2**int_info.bits_per_digit``.

   .. attribute:: int_info.sizeof_digit

      The size in bytes of the C type used to represent a digit.

   .. attribute:: int_info.default_max_str_digits

      The default value for :func:`sys.get_int_max_str_digits`
      when it is not otherwise explicitly configured.

   .. attribute:: int_info.str_digits_check_threshold

      The minimum non-zero value for :func:`sys.set_int_max_str_digits`,
      :envvar:`PYTHONINTMAXSTRDIGITS`, or :option:`-X int_max_str_digits <-X>`.
```